crt_portal/cts_forms/templates/forms/report_data.html

```diff
@@ -156,15 +157,15 @@ <h2>{{ ordered_step_names.3 }}</h2>
         <h3>{{ question.characteristics }}</h3>
         {% if report.id %}
             {% for characteristic in report.protected_class.all %}
-                <p>{{ characteristic }}</p>
+                <p>{% trans characteristic.protected_class %}</p>
```

Comment:
I think this should be taken care of here:
https://github.com/usdoj-crt/crt-portal/blob/develop/crt_portal/cts_forms/model_variables.py#L84-L103

Reply:
You're right, this one was confusing to track down....:

These translation tags were needed because the `HateCrimesandTrafficking` and `ProtectedClass` models aren't using the defined `choices` as expected. They're currently storing the  human-readable name, as lazy loaded when each object instance was created, as opposed to the value that we would expect. 

Example:
`Threatened, forced, and held...` is in the database instead of `physical_harm`. This breaks the `get_FOO_display` and default widget rendering as the value from the database isn't present in the choice list.

Visible in the admin as well within on the associated model and instance pages. 

It looks like this dates back to ec54a7645b2a8f364f78908149b31eff5b9a16d8 and impacted how some of the other templates were implemented, I'll start looking at how to remedy.